_layouts/default.html

```diff
@@ -133,8 +133,49 @@ <h2 id="title">
   {%- endif -%}
 
   {% if site.plainwhite.search %}
-  <script src="/assets/js/simple-jekyll-search.min.js"></script>
-  <script src="/assets/js/search.js"></script>
+  <script src="{{ "/assets/js/simple-jekyll-search.min.js" | relative_url }}"></script>
+  <!-- <script src="{{ "/assets/js/search.js" | relative_url }}"></script> -->
+  <script>
+	window.onload = function () {
+		var sjs = SimpleJekyllSearch({
+			searchInput: document.getElementById('searchbar'),
+			resultsContainer: document.getElementById('search-results'),
```

Comment:
Any reason for inlining this script?

Reply:
As mentioned in my original comment, it needs to be inlined so that the link on line 143 is formatted correctly with the baseurl, as liquid/Jekyll does not handle tags in external scripts.

Reply:
Did you try adding frontmatter to the script file? 
This:
```
---
---
```

Reply:
I will try that tomorrow.